src/libs/actions/IOU.ts

```diff
@@ -4044,14 +4011,12 @@ function trackExpense(
                 tag,
                 billable,
                 receipt: trackedReceipt,
-                waypoints: sanitizedWaypoints,
```

Comment:
Hi @luacmartins I'm encountering this issue https://github.com/Expensify/App/issues/53712#issuecomment-2558093360, which occurs when the waypoints parameter is missing in the `CategorizeTrackedExpense` API.

Reply:
Hey @linhvovan29546 this PR was a straight revert of another one, so maybe the issue was being fixed by the original PR? It seems like you have an ongoing discussion in your PR already, so I'll let others chime in there.